sql/derived_tables/runlist.txt

```diff
@@ -28,6 +28,7 @@ invoice_adjustments_in_addition_to.sql
 invoice_adjustments_ext.sql
 invoice_lines_adjustments.sql
 invoice_lines_fund_distributions.sql
+invoice_voucher_lines_fund_distributions
```

Comment:
This file name is missing `.sql` which causes the automated test to fail.

Reply:
Sorry about that. It's been updated.